includes/functions.php

```diff
@@ -357,3 +357,45 @@ function webmention_extract_urls( $content, $support_media_urls = false ) {
 
 	return array_filter( $urls );
 }
+
+/**
+ * Retrieve and display webmentions sent to the desingated homepage.
+ *
+ * @since NEXT
+ */
+function webmention_get_home_webmentions() {
+	$homepage = get_option( 'webmention_home_mentions', 0 );
```

Comment:
It should short circuit if 0 before the query

Reply:
Addressed in 1998135